worldengine/cli/main.py

```diff
@@ -104,13 +103,11 @@ def generate_plates(seed, world_name, output_dir, width, height,
     filename = '%s/plates_%s.png' % (output_dir, world_name)
     # TODO calculate appropriate sea_level
     sea_level = 1.0
-    draw_simple_elevation_on_file(world.elevation['data'], filename, width,
-                                  height, sea_level)
+    draw_simple_elevation_on_file(world, filename, None)
```

Comment:
Just noticed this. Even though there is technically no ocean yet, platec did run under the assumption that a certain percentage of the world was covered with ocean, that at the very least has a big influence on the resulting world. I am not sure if ```None``` is really the best choice here.

Also: You removed ```sea_level```s purpose of existence but left the variable in - this makes the pyflakes test fail. It has to be removed.